pkg/pbflow/proto.go

```diff
@@ -110,43 +108,12 @@ func FlowToPB(fr *model.Record, s *ovnobserv.SampleDecoder) *Record {
 			pbflowRecord.Xlat.DstAddr = &IP{IpFamily: &IP_Ipv4{Ipv4: model.IntEncodeV4(fr.Metrics.AdditionalMetrics.TranslatedFlow.Daddr)}}
 		}
 	}
-	if s != nil && fr.Metrics.AdditionalMetrics != nil {
-		seen := make(map[string]bool)
-		pbflowRecord.NetworkEventsMetadata = make([]*NetworkEvent, 0)
-		for _, metadata := range fr.Metrics.AdditionalMetrics.NetworkEvents {
-			var pbEvent NetworkEvent
-			if !model.AllZerosMetaData(metadata) {
-				if md, err := s.DecodeCookie8Bytes(metadata); err == nil {
-					acl, ok := md.(*ovnmodel.ACLEvent)
-					mdStr := md.String()
-					protoLog.Debugf("Network Events Metadata %v decoded Cookie: %v decoded string: %s", metadata, md, mdStr)
-					if !seen[mdStr] {
-						if ok {
-							pbEvent = NetworkEvent{
-								Events: map[string]string{
-									"Action":    acl.Action,
-									"Type":      acl.Actor,
-									"Feature":   "acl",
-									"Name":      acl.Name,
-									"Namespace": acl.Namespace,
-									"Direction": acl.Direction,
-								},
-							}
-						} else {
-							pbEvent = NetworkEvent{
-								Events: map[string]string{
-									"Message": mdStr,
-								},
-							}
-						}
-						pbflowRecord.NetworkEventsMetadata = append(pbflowRecord.NetworkEventsMetadata, &pbEvent)
-						seen[mdStr] = true
-					}
-				} else {
-					protoLog.Errorf("unable to decode Network events cookie: %v", err)
-				}
-			}
-		}
+
+	pbflowRecord.NetworkEventsMetadata = make([]*NetworkEvent, 0)
```

Comment:
nit / follow-up: avoid creating NetworkEventsMetadata if fr.NetworkMonitorEventsMD is empty?